src/components/PresenceMap/readme.md

```diff
@@ -0,0 +1,183 @@
+    text-align: center;
+    padding: 5px;
+    width: fit-content;
+    float: left;
+`;
+
+const Icon = styled(FontAwesomeIcon)`
+    vertical-align: bottom;
+    margin-top: 20%;
+`;
+
+const PresenceMapExample = () => {
+    const [showTrafficState, setShowTraffic] = useState(false);
+    const [showTransitState, setShowTransit] = useState(false);
+    const [zoomState, setZoom] = useState('auto');
+    const [centerState, setCenter] = useState('auto');
+    const [mapTypeState, setMapType] = useState({ name: 'roadmap', label: 'ROADMAP' });
+
+    const objects = [
+        {
+            position: {
+                lat: -33.836538,
+                lng: 151.1279,
+            },
+            image: "http://chart.googleapis.com/chart?chst=d_map_pin_letter&chld=xxx%7c5680FC%7c000000&.png",
+            heading: 270,
+        },
+        {
+            position: {
+                lat: -34.840538,
+                lng: 150.1279,
+            },
+            image: "http://chart.googleapis.com/chart?chst=d_map_pin_letter&chld=xxx%7c5680FC%7c000000&.png",
+            heading: 90,
+        },
+        {
+            position: {
+                lat: -35.850538,
+                lng: 151.1279,
+            },
+            image: "http://chart.googleapis.com/chart?chst=d_map_pin_letter&chld=xxx%7c5680FC%7c000000&.png",
+        },
+    ];
+
+    useEffect(() => {
+        setShowTraffic(showTrafficState);
+    }, [showTrafficState]);
+
+    useEffect(() => {
+        setShowTransit(showTransitState);
+    }, [showTransitState]);
+
+    useEffect(() => {
+        setZoom(zoomState);
+    }, [zoomState]);
+
+    useEffect(() => {
+        setCenter(centerState);
+    }, [centerState]);
+
+    const handleShowTraffic = () => {
+        setShowTraffic(!showTrafficState);
+    };
+
+    const handleShowTransit = () => {
+        setShowTransit(!showTransitState);
+    };
+
+    const handleCenterGeolocation = () => {
+        const geo_success = position => {
+            setCenter({
+                lat: position.coords.latitude,
+                lng: position.coords.longitude
+            });
+        };
+
+        const geo_error = () => {
+            alert("Sorry, no position available.");
+        };
+
+        const geo_options = {
+            enableHighAccuracy: true, 
+            maximumAge: 30000, 
+            timeout: 27000,
+        };
+
+        navigator.geolocation.getCurrentPosition(geo_success, geo_error, geo_options);
+    };
+
+    const handleMapStyle = value => {
+        setMapType(value);
+    };
+
+    return (
+        <PresenceMap
+            apiKey={LIBRARY_GOOGLE_MAPS_APIKEY}
+            objects={objects}
+            showTraffic={showTrafficState}
+            showTransit={showTransitState}
+            zoom={zoomState}
+            center={centerState}
+            type={mapTypeState.name}
+        >
+            <MenuControl
+                id="picklist-mapstyle-1"
+                style={{ float: "right" }}
```

Comment:
no style inline, use the styled inline, also I think we should avoid using the float property